src/main/java/dev/jbang/util/Util.java

```diff
@@ -72,7 +72,7 @@
 	public static final Pattern mainClassMethod = Pattern.compile(
 			"(?<=\\n|\\A)(?:public\\s)\\s*(class)\\s*([^\\n\\s]*)");
 
-	private static final List<String> EXTENSIONS = asList(".java", ".jsh", ".kt");
+	private static final List<String> EXTENSIONS = asList(".java", ".jsh", ".kts", ".groovy");
```

Comment:
This are 2 unrelated changes.

Reply:
Kts!= kt. 

Kt is compiled first. Kts is not. 

Jbang does not support kts style scripts. At least not in its current shape.

Reply:
yes, it's my fault.  I'm working on kts support for jbang, and sorry for a mistake introduced.

Reply:
It's OK. You just have to keep the changes separate from each other 😄